frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -287,16 +287,34 @@ def build_task_canceled(task_id):
 @backend_ns.route("/pending-actions/")
 def pending_actions():
     'get the list of actions backand should take care of'
+    waiting_actions = actions_logic.ActionsLogic.get_waiting()
+    busy_namespaces = set()
+    for action in waiting_actions:
+        if (
+            action.object_type == ActionObjectTypeEnum.COPR
+            and action.action_type == ActionTypeEnum("delete")
```

Comment:
Ah, I now see - we have to do this complicated `object_type && action_type` condition :-( this is IMO clear sign that we shouldn't implement abstraction for `ActionObjectTypeEnum`, but rather remove it in the future.  I'd simply keep the string representation for now, drop there some "TODO" item in models.py that this should go away (and perhaps solve this in a separate PR).